CHANGES/4414.doc

```diff
@@ -0,0 +1 @@
+Clarify that `GracefulExit` needs to be handled in `AppRunner` and `ServerRunner` when using `handle_signals=True`.
```

Comment:
This should use RST syntax, not Markdown: https://aiohttp--7043.org.readthedocs.build/en/7043/changes.html#improved-documentation.

Reply:
Should be fixed in [e075e56](https://github.com/aio-libs/aiohttp/pull/7043/commits/e075e56959937c454ec26d08a003ad766e604c0d).